### PR TITLE
add brew formula for cable-bible

### PR DESCRIPTION
**Pull request by MIAPtech**
once [pull request](https://github.com/amiaopensource/cable-bible/pull/23) to open cable-bible locally via CLI is approved, this formula will allow for install/update via homebrew and linuxbrew

### CODE REVIEW

**Comment by privatezero** [comment]
yay! LGTM!

File: cable-bible.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CableBible < Formula
+  desc "A comprehensive guide to cables and connectors used in audiovisual archiving"
+  homepage "https://github.com/amiaopensource/cable-bible"
+  url "https://github.com/amiaopensource/cable-bible/archive/v2017-02-27.tar.gz"
+  sha256 "9fd5a792dc0973721abe08963775072462ce5ee0f1c115b6cfb7ca38c8420902"
+
+  def install
+  bin.install "cable-bible"
```
Comment (retokromer) [inline cable-bible.rb]: Please indent this and the following line.

File: cable-bible.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  prefix.install "css", "images", "js", "README.md", "code_of_conduct.md", "index.html", "params.json", "pinouts.md"
+  end
+
```
Comment (retokromer) [inline cable-bible.rb]: Please delete this empty line.

**Comment by MIAPtech** [comment]
edited!

**Comment by retokromer** [comment]
Excellent, thank you!

**Comment by retokromer** [comment]
Oops! It gives:
```
Error: No such file or directory - cable-bible
```

**Pull request merged**